src/test/java/com/squareup/javapoet/TypeSpecTest.java

```diff
@@ -413,9 +415,9 @@
         .addTypeVariable(t)
         .addTypeVariable(p)
         .addSuperinterface(Types.parameterizedType(Comparable.class, p))
-        .addField(FieldSpec.of(t, "label"))
-        .addField(FieldSpec.of(p, "x"))
-        .addField(FieldSpec.of(p, "y"))
```

Comment:
I'm a sucker for the consistency in builder use (and these helper methods on parent specs), but I can't decide if this is tailoring too much to the test case uses. You can see a counter-example to the current PR's change above where you're now forced to use the builder to create a simple `ParameterSpec`.

Most code that is going to be using JavaPoet will be fraught with conditionals, loops, and helper methods that return objects. You can see this in the PR you made to the you-know-what project to make the switch to JavaPoet using this PR.

I'm fine with removing the helpers. But let's not completely write them off yet as we start to actually play with this more.